examples/onnxrt/nlp/huggingface_model/code_detection/quantization/ptq_dynamic/requirements.txt

```diff
@@ -0,0 +1,11 @@
+torch
+transformers
+accelerate
+onnx
+onnxruntime
+coloredlogs
+sympy
+onnxruntime-extensions; python_version < '3.10'
```

Comment:
onnxruntime-extensions is available for python 3.10 now. Please update < '3.10' to < '3.11'

Reply:
Updated by [345c1b5](https://github.com/intel/neural-compressor/pull/1121/commits/345c1b5c62ae066fbc88596ea7737988f927f8dd).